ci-templates/stages.yml

```diff
@@ -32,7 +32,8 @@ stages:
 
           - task: CacheBeta@0
             inputs:
-              key: maven | bom/runtime/pom.xml   #if we attempt to use all poms then when they get copied to target everything breaks. This should be good enough, it does not need to be perfect
+              # the number below is a cache version when we want to force a cache refresh
+              key: maven | "2" | bom/runtime/pom.xml   #if we attempt to use all poms then when they get copied to target everything breaks. This should be good enough, it does not need to be perfect
```

Comment:
Does this mean that we will need to increment the version number somehow?

Reply:
No, it's only because I really wanted to go from a clean cache and I didn't find another way to do it.

So basically, you just have to increment when you have buggy artifacts in your cache and you want to start from scratch.

Reply:
That's what I thought, but just wanted to be sure